### DISCUSSION TITLE
Possible to install apps without creating the shims?

### DISCUSSION THREAD

**Discussion by klodoma** [ScoopInstaller/Scoop#6104] [Help and Questions]
Is it possible to install an app but skip the shim creation?

I try to install the different versions of php, but skip the creation of `php.exe` in `shims` folder

```
scoop install versions/php83
scoop install versions/php82
```

**Comment by HUMORCE** [comment]
Skipping shim creation is not supported for now, but you can use `scoop shim` to manage shims.
